### ISSUE TITLE
Problem with transcript_quantification

### ISSUE THREAD

**Issue by kir1to455** [ablab/IsoQuant#39]
Hi，andrewprzh
I've been running isoquant succesfully so far. Looks quite good.
Now I have 13 input samples(7 Cancer , 6 Tissue), I want to get the aggregate GTF, and it work on actually and succesfully.

However, I found that the output file  "Ca_Ti_MergeV3.transcript_model_grouped_counts.tsv" which nic/nnic transcripts with all samples transcript_quantification is 0. What's the meaning of these "all zeros" nic, nnic? And in which sample could I find this nic/nnic ? 
![2a7c26f8b65e39dc3667b38f0d004f9](https://user-images.githubusercontent.com/87375686/189659291-e338cf15-88ad-4776-a160-0416742d58a8.png)

Here is my command line:
`#!/bin/bash
isoquant.py -d nanopore \
 --stranded forward \
 -t 40 \
 --bam_list /data2/HJL/BN200908NJ02S02N1/listV3.txt \
 --read_group file_name \
 -l Ca_Ti_Merge \
 --reference /data2/HJL/ref/GCA_000001405.15_GRCh38_no_alt_analysis_set.fa \
 --genedb /data2/HJL/ref/gencode.v41.annotation.gtf \
 --complete_genedb \
 --output Ca_Ti_MergeV3`

listV3.list as following:
`/aux/DYJ_Cancer.bam
/aux/Y21_Cancer.bam
/aux/Y47_Cancer.bam
/aux/Y75_Cancer.bam
/aux/Y76_Cancer.bam
/aux/LNCaP.bam
/aux/PC3.bam
/aux/DYJ_Tissue.bam
/aux/Y21_Tissue.bam
/aux/Y75_Tissue.bam
/aux/Y76_Tissue.bam
/aux/Y47_Tissue.bam
aux/Y34_Tissue.bam`

Thanks for your help!
Best,
Kir1to455

**Comment by andrewprzh** [comment]
Dear @kir1to455 

Thank you for the report!

Can any of those "all-zero" transcripts be found in GTF file? 
My guess is that "all-zero" transcripts are those that did not pass filters and did not make it to final GTF, but still were printed in the count table. If so, you can simply ignore those. I think this is a minor bug in the output function, will fix!

Best
Andrey

**Comment by kir1to455** [comment]
Hi, andrewprzh

Great and many thanks for the quick reply.
I found that these "all-zero" transcripts  not in GTF file, it can be completely ignored.

But I have a new questions about the counts.tsv output file,  I found that these counts are followed by decimals.
Generally speaking, these counts are counts of reads, which should be integers.
![image](https://user-images.githubusercontent.com/87375686/189898886-adccbce0-4449-41d9-b5f8-cb1c5775ea2a.png)
Why do decimals appear in these counts file?
![image](https://user-images.githubusercontent.com/87375686/189899260-28d8279f-fce8-4a38-9e71-b8468601d8de.png)


Thanks for your help!
Best,
Kir1to455

**Comment by andrewprzh** [comment]
Yes, all-zeros can be ignored for now, will fix later - thanks for the report!

The second question is answered here https://github.com/ablab/IsoQuant/issues/40, sorry for the delayed response.
